app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PasswordsController < ApplicationController
+  before_action :redirect_users_to_home
```

Comment:
I think either the first or last of those are good choices. Naming can be very difficult!